nuitka/build/static_src/HelpersTypes.c

```diff
@@ -280,36 +288,73 @@ static PyTypeObject *_getTypeGenericAliasType(void) {
 
     if (type_generic_alias_type == NULL) {
 
-        PyObject *typing_module = PyImport_ImportModule("_typing");
-        CHECK_OBJECT(typing_module);
+        PyObject *types_module = PyImport_ImportModule("types");
```

Comment:
For most of these cases, there is something like `IMPORT_HARD_TYPES` that is cached and all, and preferable, changing that over.